internal/core/core_test.go

```diff
@@ -515,12 +516,73 @@ func TestFenceBorders(t *testing.T) {
 		interpolationMethod, _ := GetInterpolationMethod("linear")
 		handle, _ := NewVDSHandle(well_known)
 		defer handle.Close()
-		_, err := handle.GetFence(testcase.coordinate_system, testcase.coordinates, interpolationMethod)
+		_, err := handle.GetFence(testcase.coordinate_system, testcase.coordinates, interpolationMethod, nil)
 
 		require.ErrorContainsf(t, err, testcase.err, "[case: %v]", testcase.name)
 	}
 }
 
+func TestFenceBordersWithFillValue(t *testing.T) {
```

Comment:
I think combining both tests will increase the complexity of the test case. I will change the order of the commit before merging so that it makes more sense.